impeller/entity/contents/content_context.cc

```diff
@@ -543,8 +543,8 @@ fml::StatusOr<RenderTarget> ContentContext::MakeSubpass(
   return subpass_target;
 }
 
-std::shared_ptr<Tessellator> ContentContext::GetTessellator() const {
-  return tessellator_;
+Tessellator& ContentContext::GetTessellator() const {
```

Comment:
This should be `const Tessellator& ContentContext::GetTessellator() const` or `Tessellator& ContentContext::GetTessellator()`

https://google.github.io/styleguide/cppguide.html#Use_of_const

Reply:
tessellator methods are non const because it stores an allocation arena we use.

Reply:
So `Tessellator& ContentContext::GetTessellator()`?

Reply:
Yeah?

Reply:
That's not what's there, you have a const method returning a non-const ref =)

Reply:
> Declare methods to be const unless they alter the logical state of the object (or enable the user to modify that state, e.g., by returning a non-const reference, but that's rare), or they can't safely be invoked concurrently.

Reply:
I don't think that is what that means. The ContextVK is const, the tessellator is not. same with the host buffer?

Reply:
Constness should be transitive where if X is a aggregate of Y and Z and you have a const X, you should only be able to get a const Y and const Z.

There are already instances where this isn't the case here and admittedly Context is an odd grab bag of state.  I was suggesting it if it's easy to do, otherwise this is fine since it doesn't regress what was already there.

Reply:
If that is an important policy to enforce, then it really needs to enforced via lints or the compiler.